unittests/test_fixtures.py

```diff
@@ -202,7 +202,6 @@ class FooB(rfm.RegressionTest):
 
     class Bar(rfm.RegressionTest):
         f = fixture(FooB, variables={'z': 5})
-        w = fixture(FooB, action='join')
```

Comment:
I wonder why we are removing the unit test here...
Don't we need a unit test for this too?

Reply:
The fact that this worked was technically a bug of the `FixtureProxy` and it shouldn't have been there, since join fixtures are not proxied as they don't parameterise the parent test. Join fixtures are also accessed as an iterable, so you should not be able to do `self.w.y`. The fact that this worked was that `FooB` had only one parameter value. If it had two, the test would have failed. Now we are consistent.